src/components/JumpBar.js

```diff
@@ -86,9 +86,9 @@ const JumpBar = ({items, isFetching, clear, search}) => {
 
 function getPath(type, id) {
   switch (type) {
-    case 'container':  return `/containers/${id}`;
-    case 'sample':     return `/samples/${id}`;
-    case 'individual': return `/individuals/${id}`;
+    case 'container':  return `/containers/${id}?nested=True`;
+    case 'sample':     return `/samples/${id}?nested=True`;
+    case 'individual': return `/individuals/${id}?nested=True`;
```

Comment:
Aren't these front-end routes? Why is `nested` needed here?

Reply:
adding this got rid of some errors from what i recall, they will be going anyway since nesting is going away...